conpot/conpot.cfg

```diff
@@ -39,6 +39,9 @@ host = taxiitest.mitre.org
 port = 80
 inbox_path = /services/inbox/default/
 use_https = False
+
+[stix]
+port_path_list = ["//conpot_template/protocols/modbus/@port","//conpot_template/protocols/snmp/@port","//conpot_template/protocols/http/@port","//conpot_template/protocols/s7comm/@port"]
```

Comment:
maybe split it to more lines, and add \ to the end of line ? :)

like
[ \
    "//conpot_template/protocols/modbus/@port", \
    "//conpot_template/protocols/snmp/@port", \
    "//conpot_template/protocols/http/@port", \
    "//conpot_template/protocols/s7comm/@port" \
]